naas/models/account_settings.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from requests import AccountSettings
```

Comment:
@arthuroe - this gave me an error, as there is no `AccountSettings` inside of `requests`.